src/mono-runtimes/mono-runtimes.targets

```diff
@@ -301,18 +301,18 @@
         Directories="$(MonoSourceFullPath)\sdks\out"
     />
     <DownloadUri
-        SourceUris="https://xamjenkinsartifact.blob.core.windows.net/mono-sdks/$(_MonoArchiveName).tar.gz"
-        DestinationFiles="$(AndroidToolchainCacheDirectory)\$(_MonoArchiveName).tar.gz"
+        SourceUris="https://xamjenkinsartifact.blob.core.windows.net/mono-sdks/$(_MonoArchiveName).zip"
+        DestinationFiles="$(AndroidToolchainCacheDirectory)\$(_MonoArchiveName).zip"
         ContinueOnError="True"
     />
     <Exec
-        Condition=" Exists('$(AndroidToolchainCacheDirectory)\$(_MonoArchiveName).tar.gz') "
-        Command="tar -xvf $(AndroidToolchainCacheDirectory)\$(_MonoArchiveName).tar.gz"
+        Condition=" Exists('$(AndroidToolchainCacheDirectory)\$(_MonoArchiveName).zip') "
+        Command="unzip $(AndroidToolchainCacheDirectory)\$(_MonoArchiveName).zip"
```

Comment:
Paths should be `&quot;`ed in case they contain spaces.

Additionally, this could probably use the `<UnzipDirectoryChildren/>` task in https://github.com/xamarin/xamarin-android/blob/f3e37c6b1d77205e8d8cde7d6e866f8d3f726621/build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/UnzipDirectoryChildren.cs.

This would have the added benefit of allowing the `_DownloadArchive` target to run on Windows.